### PR TITLE
markdown-it-js : initial integration

### PR DESCRIPTION
**Pull request by adoxalim**
initial integration for javascript **markdown-it** library

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
We approve of adding this project. Does upstream agree to it?

**Comment by adoxalim** [comment]
@jonathanmetzman I came here after found, disclosure and fix a bug I create this PR. So they know about it, and they agree with it.

**Comment by jonathanmetzman** [comment]
> @jonathanmetzman I came here after found, disclosure and fix a bug I create this PR. So they know about it, and they agree with it.

I want this integrated but I'm not sure I understand this comment. But if you can say that you told them you are doing an OSS-Fuzz integration and they were happy about it, I will merge.

**Comment by adoxalim** [comment]
:-) Yes I said that.

File: projects/markdown-it-js/project.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+homepage: https://github.com/markdown-it/markdown-it
+language: javascript
+primary_contact: adoxalim@gmail.com
```
Comment (oliverchang) [inline projects/markdown-it-js/project.yaml:3]: Could we please make sure to get an upstream maintainer involved here?

**Pull request merged**